src/config.rs

```diff
@@ -519,11 +519,17 @@ impl Context {
 
         // Default values
         let val = match key {
-            Config::ConfiguredInboxFolder => Some("INBOX"),
-            Config::DeleteServerAfter => match Box::pin(self.is_chatmail()).await? {
-                false => Some("0"),
-                true => Some("1"),
+            Config::BccSelf => match Box::pin(self.is_chatmail()).await? {
+                false => Some("1"),
+                true => Some("0"),
             },
+            Config::ConfiguredInboxFolder => Some("INBOX"),
+            Config::DeleteServerAfter => {
+                match Box::pin(self.get_config(Config::BccSelf)).await?.as_deref() {
+                    Some("0") => Some("1"),
```

Comment:
A check for `IsChatmail` is forgotten

Reply:
Though it's an interesting question -- is leaving emails on the server useful w/o `BccSelf`? Probably not for most email providers because outgoing messages would be missing anyway. But to keep the current behaviour a check for `IsChatmail` should be added. Also other MUAs don't delete emails from the server by default even if the server doesn't save sent messages to some sentbox (like Gmail does)